plugins/modules/identity/sap_user.py

```diff
@@ -0,0 +1,510 @@
+'''
+
+RETURN = r'''
+msg:
+  description: A small execution description.
+  type: str
+  returned: always
+  sample: 'User ADMIN created'
+out:
+  description: A more detailed description.
+  type: list
+  elements: dict
+  returned: on success
+  sample: [...,{
+              "RETURN": [
+            {
+                "FIELD": "BNAME",
+                "ID": "01",
+                "LOG_MSG_NO": "000000",
+                "LOG_NO": "",
+                "MESSAGE": "User ADMIN created",
+                "MESSAGE_V1": "ADMIN",
+                "MESSAGE_V2": "",
+                "MESSAGE_V3": "",
+                "MESSAGE_V4": "",
+                "NUMBER": "102",
+                "PARAMETER": "",
+                "ROW": 0,
+                "SYSTEM": "",
+                "TYPE": "S"
+            }
+          ],
+          "SAPUSER_UUID_HIST": []}]
+'''
+from ansible.module_utils.basic import AnsibleModule, missing_required_lib
+import traceback
+import datetime
+try:
+    from pyrfc import Connection
+except ImportError:
+    HAS_PYRFC_LIBRARY = False
+    PYRFC_LIBRARY_IMPORT_ERROR = traceback.format_exc()
+else:
+    HAS_PYRFC_LIBRARY = True
+
+
+def add_to_dict(target_dict, target_key, value):
+    # Adds the given value to a dict as the key
+    # check if the given key is in the given dict yet
+    if target_key in target_dict:
+        return False
+    target_dict[target_key] = value
+    return True
+
+
+def call_rfc_method(connection, method_name, kwargs):
+    # PyRFC call function
+    return connection.call(method_name, **kwargs)
+
+
+def build_rfc_user_params(username, firstname, lastname, email, raw_password,
+                          useralias, user_type, raw_company, user_change):
+    """Creates RFC parameters for Creating users"""
+    # define dicts in batch
+    params = dict()
+    address = dict()
+    password = dict()
+    alias = dict()
+    logondata = dict()
+    company = dict()
+    # for change parameters
+    addressx = dict()
+    passwordx = dict()
+    logondatax = dict()
+    companyx = dict()
+    force = bool()
+    # define username
+    add_to_dict(params, 'USERNAME', username)
+    # define Address
+    add_to_dict(address, 'FIRSTNAME', firstname)
+    add_to_dict(address, 'LASTNAME', lastname)
+    add_to_dict(address, 'E_MAIL', email)
+    # define Password
+    add_to_dict(password, 'BAPIPWD', raw_password)
+    # define Alias
+    add_to_dict(alias, 'USERALIAS', useralias)
+    # define LogonData
+    add_to_dict(logondata, 'GLTGV', datetime.date.today())
+    add_to_dict(logondata, 'GLTGB', '20991231')
+    add_to_dict(logondata, 'USTYP', user_type)
+    # define company
+    add_to_dict(company, 'COMPANY', raw_company)
+    #
+    params['LOGONDATA'] = logondata
+    params['ADDRESS'] = address
+    params['COMPANY'] = company
+    params['ALIAS'] = alias
+    params['PASSWORD'] = password
+    # add change if user exists
+    if user_change and force:
+        add_to_dict(addressx, 'FIRSTNAME', 'X')
```

Comment:
force is defined as bool() in 276 and didn't change until this point. So why not just 
```
if user_change:
...
```

Reply:
user defined